src/sentry/static/sentry/app/components/smartSearchBar/index.jsx

```diff
@@ -482,7 +483,16 @@ class SmartSearchBar extends React.Component {
       });
 
       try {
-        const values = await this.props.onGetTagValues(tag, query);
+        const {location} = this.context.router;
+        const endpointParams = {};
+
+        if (location && location.query){
+          endpointParams.statsPeriod = location.query.statsPeriod ? location.query.statsPeriod : "14d";
```

Comment:
@wmak we have a really nice function to derive `statsPeriod`, `start`, and `end` values: https://github.com/getsentry/sentry/blob/16452ca994e81e33e108c4dafa356b6bc01d0994/src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/getParams.tsx#L109-L142

usage is something like:

```js
const {start, end, statsPeriod} = getParams(location.query);
```

Reply:
🙏 added!